libraries/AC_InputManager/AC_InputManager_Heli.h

```diff
@@ -46,8 +49,17 @@ class AC_InputManager_Heli : public AC_InputManager {
         bool use_stab_col; // 1 if we should use Stabilise mode collective range, 0 for Acro range
     } _im_flags_heli;
 
-    //  factor used to smoothly ramp collective from Acro value to Stab-Col value
-    float _stab_col_ramp = 0;
+    // init transition ramp
+    bool init_transition_ramp = false;
+
+    // current flight mode collective output
+    float _new_flightmode_col_output = 0;
+
+    // previous flight mode collective output
+    float _old_flightmode_col_output = 0;
+
+    // ramp factor from previous mode to current mode collective output
+    float _ramp = 0;
```

Comment:
I believe we try and maintain a pattern of not initialising a variable in the header file if can help it.  As you set the ramp value in the heli_init() function in heli.cpp this initialisation is not needed.
```suggestion
    float _ramp;
```